readme.md

```diff
@@ -53,12 +67,12 @@ Seriously, commit now. It will be easy to fix any devise issues if you have a co
 ```
 $ rails g devise User
 ```
-If you have an existing User model, Devise should build on the functionality. But, if you're getting migration errors, it may be easier to recreate the model using Devise and add your functionality back.
+If you have an existing User model, Devise _should_ build on the functionality. But, if you're getting migration errors, it may be easier to recreate the model using Devise and add your functionality back.
 
```

Comment:
I think here it might be nice to just put in a blurb, about what was created. maybe even dive into the migration file and check that out. Also, because devise hides a lot of shit, might be cool to take a look at the devise source. Like compare rake routes with the registrations controller